package.json

```diff
@@ -27,9 +28,12 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
+    "test:jest": "jest",
+    "e2e": "BROWSER=none yarn start & cypress run",
```

Comment:
Mettre `e2e:run` pour que ça matche avec ton mail (sinon `error Command "e2e:run" not found.`)

cf 
```
Tu peux run les tests via : 
yarn e2e:run 
--> run les tests en console (adapté pour CI)

ou sinon

yarn e2e:ui
-> affiche les test runnable via une UI Electron
(adapté pour du test authoring)
```